content/blog/how-elevate-supports-business-needs.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "How ELevate supports business needs"
+type: blog
+author: 
+ name: "David der Nederlanden"
+ bio: "Linux Cloud Architect at Bizway"
+ image: /users/ddernederlanden.jpg
+date: '2024-05-22'
+images:
+  - /blog-images/2024/2024-04-23-centos6-to-centos7.png
+post:
+    title: "How ELevate supports business needs"
+    image: /blog-images/2024/2024-04-23-centos6-to-centos7.png
+---
+
+# How Elevate supports business needs
+While we've been busy upgrading all our CentOS 7 servers to AlmaLinux 8 and higher, the ELevate Project now has the abillity to upgrade CentOS 6 servers too!
```

Comment:
```suggestion
# How Elevate supports business needs
As long-time users of AlmaLinux, we were already familiar with Project ELevate for upgrading CentOS 7, but just a few weeks ago we got some exciting news. While we've been busy upgrading all our CentOS 7 servers to AlmaLinux 8 and higher, the ELevate Project now has the ability to upgrade CentOS 6 servers too!
```
This just felt like it needed a little more context for readers.

Reply:
Agreed!